helpers/common.go

```diff
@@ -72,26 +83,44 @@ func UnameMachine() (string, error) {
 // 4.18.0-305.7.1.el8_4.centos.x86_64 (centos)
 // 4.18.0-305.7.1.el8_4.centos.plus.x86_64 (centos + plus repo)
 // 5.13.13-arch1-1 (archlinux)
-//
-func CompareKernelRelease(base, given string) int {
+func CompareKernelRelease(base, given string) (int, error) {
 	b := strings.Split(base, "-") // [base]-xxx
 	b = strings.Split(b[0], ".")  // [major][minor][patch]
 
+	if len(b) == 2 {
+		b = append(b, "0")
+	}
+	if len(b) != 3 {
+		return KernelVersionInvalid, fmt.Errorf("invalid base kernel version format: %s", base)
+	}
```

Comment:
I know it's not common but what if `base`/`given` are just "5". Could we interpret them as 5.0.0?

```suggestion
	b := strings.Split(base, "-") // [base]-xxx
	b = strings.Split(b[0], ".")  // [major][minor][patch]
	
	if len(b) == 0 {
		return KernelVersionInvalid, fmt.Errorf("invalid base kernel version format: %s", base)
	}

	for len(b) < 3 {
		b = append(b, "0")
	}
```

WDYT?